qa-tests-backend/test-images/trigger-policy-violations/most/Dockerfile

```diff
@@ -8,9 +8,14 @@ WORKDIR /
 # RUN echo "a secret" > /run/secrets/asecret
 # VOLUME /run/secrets
 
+RUN yum update -y
+
 # For: "Shell Spawned by Java Application"
 RUN yum -y install csh
```

Comment:
Just use bash in the `run-csh.sh` script (probably change the name of the file and be sure to switch that file to use bash instead of csh)

Reply:
Removing `csh` would allow us to ubi9 as well, but it looks like it was initial: https://github.com/stackrox/stackrox/blob/master/qa-tests-backend/test-images/trigger-policy-violations/most/trigger-violations.sh#L61-L62

Maybe bash will suffice?

Reply:
yes remove csh and used bash in the script